src/prefect/deployments/runner.py

```diff
@@ -264,19 +264,31 @@ def from_flow(
             version=version,
         )
 
-        # TODO: better error messages with doc links
         if not deployment.entrypoint:
+            no_file_location_error = (
+                "Flows defined interactively cannot be deployed. Check out the"
+                " quickstart guide for help getting started:"
+                " https://docs.prefect.io/latest/getting-started/quickstart."
```

Comment:
```suggestion
                " https://docs.prefect.io/latest/getting-started/quickstart"
```
i just know people are going to accidentally get that trailing period when they copy / paste haha